src/app/widget-display/widget-display.component.html

```diff
@@ -0,0 +1,32 @@
+<mat-form-field class="search-field">
+    <input type="text" placeholder="Search City" aria-label="Search City" matInput [formControl]="myControl"
+        [matAutocomplete]="auto">
+    <button mat-button (click)="getWeather(myControl.value)">Get Weather</button>
+    <mat-autocomplete #auto="matAutocomplete">
+        <mat-option *ngFor="let option of filteredOptions | async" [value]="option">
+            {{ option }}
+        </mat-option>
```

Comment:
try to use for() {} it's the new built-in syntax for control flow
https://angular.io/api/core/for